cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc

```diff
@@ -22,9 +22,9 @@ The components of the database privilege commands are:
 ** `CREATE NEW [PROPERTY] NAME` - allows property names to be created, so that nodes and relationships can have properties with these names assigned.
 ** `NAME [MANAGEMENT]` - allows all of the name management capabilities: node labels, relationship types, and property names.
 ** `ALL [[DATABASE] PRIVILEGES]` - allows access, index, constraint, and name management for the specified database.
-** `SHOW TRANSACTION {* | user[, ...]}` -  allows listing transactions and queries for the specified users on the specified database.
-** `TERMINATE TRANSACTION {* | user[, ...]}` - allows ending transactions and queries for the specified users on the specified database.
-** `TRANSACTION [MANAGEMENT] {* | user[, ...]}` - allows listing and ending transactions and queries for the specified users on the specified database.
+** `SHOW TRANSACTION (* | user[, ...])` -  allows listing transactions and queries for the specified users on the specified database.
+** `TERMINATE TRANSACTION (* | user[, ...])` - allows ending transactions and queries for the specified users on the specified database.
+** `TRANSACTION [MANAGEMENT] (* | user[, ...])` - allows listing and ending transactions and queries for the specified users on the specified database.
```

Comment:
Should probably be `({* | user[, ...]})`.

@OliviaYtterbrink we should probably decide how we want to do in these lists. For transaction it looked this way, for `SET/REMOVE LABEL` is just says `labels` and `READ/MATCH/SET PROPERTY` had `{props}`, neither which is fully correct either since they too can have `*` or a list (and property has `{}` as part of syntax while labels don't). Not sure if there are more places but we should decide how we want to have them in the lists (easiest is to just have the privilege name and not include the user/label/property list since they might get chunky, properties for example would be `"{" {* | prop[, ...]} "}"`)

Reply:
`MERGE` privileges also have a property list or * in the syntax but not in the privilege list, there it only has the name.

Reply:
See also my other comment about this, https://github.com/neo4j/neo4j-documentation/pull/762#discussion_r528561758